src/pytom_tm/tmjob.py

```diff
@@ -809,8 +869,33 @@ def start_job(
 
         # if tilt angles are provided we can create wedge filters
         if self.tilt_angles is not None:
-            # for the tomogram a binary wedge is generated to explicitly set the
-            # missing wedge region to 0
+            if self.tilt_weighting and self.defocus_handedness != 0:
+                # adjust ctf parameters for this specific patch in the tomogram
+                full_tomo_center = np.array(self.tomo_shape) / 2
+                patch_center = (
+                    np.array(self.search_origin) + np.array(self.search_size) / 2
+                )
+                relative_patch_center_um = (patch_center - full_tomo_center) * 1e-4
+                defocus_offsets = get_defocus_offsets(
+                    relative_patch_center_um[0],
+                    relative_patch_center_um[2],
+                    self.tilt_angles,
+                    angles_in_degrees=True,
+                    invert_handedness=self.defocus_handedness < 0,
+                )
+                for ctf, defocus_shift in zip(self.ctf_data, defocus_offsets):
+                    ctf["defocus"] = ctf["defocus"] + defocus_shift * 1e-6
+                logging.debug(
+                    f"Patch center (nr. of voxels): "
```

Comment:
no an fstring needed
```suggestion
                    "Patch center (nr. of voxels): "
```